frontend/uno.config.ts

```diff
@@ -31,7 +38,8 @@ export default defineConfig({
         // the default
         /\.(vue|svelte|[jt]sx|mdx?|astro|elm|php|phtml|html)($|\?)/,
         // include js/ts files
-        "components/ui/**/*.{js,ts}"
+        "components/ui/**/*.{js,ts}",
```

Comment:
You don't have to duplicate that. `"components/**/*.{js,ts}"` already includes all subdirectories